webapp/test/testSetup.js

```diff
@@ -0,0 +1,16 @@
+import { createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import VTooltip from 'v-tooltip'
+import Styleguide from '@human-connection/styleguide'
+import BaseComponents from '~/plugins/base-components'
+import Filters from '~/plugins/vue-filters'
+import InfiniteLoading from '~/plugins/vue-infinite-loading'
+
+global.localVue = createLocalVue()
+
+global.localVue.use(Vuex)
+global.localVue.use(VTooltip)
+global.localVue.use(Styleguide)
+global.localVue.use(BaseComponents)
+global.localVue.use(Filters)
+global.localVue.use(InfiniteLoading)
```

Comment:
really great that you found this!! and I like your explanation for why it was added to this `PR`, and for taking the time to explain, in general!!
Really nice! it makes it easier for your `PR` reviewer when you explain how it is related to the changes you introduce in your `PR`, and also helps the `PR` creator to question whether it's code creep and should be in a separate `PR`.
It's not always cut and dry, but this is a really good habit for all of us to get into!
Thanks ;)